src/PubSub/Topic.php

```diff
@@ -98,10 +98,11 @@ public function __construct(
         $this->info = $info;
 
         // Accept either a simple name or a fully-qualified name.
-        if ($this->isFullyQualifiedName('topic', $name)) {
+        try {
```

Comment:
Why does the similar condition in `Subscription` check whether or not the name is null, but this doesn't?